src/test/java/net/masterthought/cucumber/ConfigurationTest.java

```diff
@@ -420,4 +421,52 @@ public void setNotFailingStatuses_SkipsNullValues() {
         // then
         assertThat(configuration.getNotFailingStatuses()).containsExactly(notFailingStatus);
     }
+
+    @Test
+    public void chartsNotToRender_FEATURES_STATISTICS() {
+
+        // given
+        Configuration configuration = new Configuration(outputDirectory, projectName);
+        assertThat(configuration.containsChartToRender(ChartType.FEATURES_STATISTICS)).isTrue();
```

Comment:
assertions must be located to // then section

Reply:
+1

Reply:
Removed asserts from //then sections